### PR TITLE
add Rust logo

### PR DESCRIPTION
**Pull request by rosslh**
<s>Circle-S icon retrieved from Soylent's [Twitter](https://twitter.com/soylent) and [Facebook](https://facebook.com/Soylent) pages.</s>

### CODE REVIEW

**Comment by davidklebanoff** [comment]
Hi @rosslh, thanks for the contribution!

Unfortunately the circle looks quite rough, how did you go about extracting the icon? Did you use a PNG to SVG converter?

At quick glance I did not see any official SVGs of their icon logo. Just the main logo on their website: https://www.soylent.com/ - I suppose someone could re-create the icon logo fairly easily by extracting the s from the official logo and re-drawing the circle around it to match... but that's no small task.

**Comment by rosslh** [comment]
@davidklebanoff ok I'll look into fixing that, in the meantime I'll use this pull request to add the Rust logo, related to #469

**Comment by davidklebanoff** [comment]
Ok. Happy to have you helping out! In the future I recommend making a branch per icon on your forked repos so you can easily have multiple pull requests at once without conflicts. :) 

As for the rust icon, it looks like you're missing the `role="img"` attribute we include between the `aria-labelledby="simpleicons-rust-icon"`  and `viewBox="0 0 24 24"` attributes.

Additionally, the quality looks lower to me than we would like. For instance, the circles on the gear aren't very smooth, an artifact of compression I would assume. Can you walk me through how you prepared/compressed the file? If you followed our [contribution guidelines](https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md) and used SVGO with the `-p 3` flag, we may have to reconsider our recommended precision level.

**Comment by rosslh** [comment]
Here's the process I went through:
1. download logo from https://upload.wikimedia.org/wikipedia/commons/d/d5/Rust_programming_language_black_logo.svg
2. resize to 24x24 in vectr.com
3. run through svgomg default settings, compression level 3 (you're right, this is where the quality decreases)
4. refactor manually in text editor to specifications laid out in contributing.md (I forgot about role="img" in this case)

Here is the logo at each stage of this process: [rust.zip](https://github.com/simple-icons/simple-icons/files/1529584/rust.zip)

**Comment by davidklebanoff** [comment]
Thanks for the detailed write up! It appears that step 2 is actually the culprit. The issues I'm seeing, the "low polygon" curves on the circles are visible in rust2.svg that you included in the .zip. Additionally, uploading rust1.svg to [SVGOMG](file:///Users/dave/Downloads/rust/rust1.svg) and playing with the precision shows that lowering the precision to something like 1 introduces artifacts, but not like the ones we're seeing here.

So to sum up, I would avoid using vectr.com as it's doing some weird compression that's trashing the icon quality. I personally use [icomoon](https://icomoon.io/app/) to do quick online resizing.

**Comment by rosslh** [comment]
Thanks for the help @davidklebanoff! I've resized using icomoon and the icon looks a lot better.

**Comment by davidklebanoff** [comment]
Awesome, looks much better. Thanks for the contribution.

In the future, please try to use official websites to source the SVGs. This time you used wikimedia, which works out in this case because they pulled it from the official website (you can see in the attribution), but just know official sources are much preferred when available.
